src/jit/lsra.h

```diff
@@ -647,6 +647,9 @@ class LinearScan : public LinearScanInterface
     // Used by Lowering when considering whether to split Longs, as well as by identifyCandidates().
     bool isRegCandidate(LclVarDsc* varDsc);
 
+    // Return the registers killed by the given tree node.
+    regMaskTP getKillSetForNode(Compiler* compiler, GenTree* tree);
```

Comment:
Where is this used outside of the register allocator? I presume that's why it is split out, but it is rather confusing that there are two methods, one of which calls the other. This could use some additional comments, both here and at the definitions.